mocks/mocks.py

```diff
@@ -21,12 +21,17 @@
 from mocks.utils import load_yaml_config
 
 
+def load_configuration(config_location='config_example.yaml'):
```

Comment:
Based on Anthony's advice, I tried to create a dictionary and pass it through other functions instead of kwargs.

Reply:
This function is pointless, the effect of having this is just to create an alternative name for `mocks.utils.yaml_load_config`. You could achieve exactly the same effect with `from mock.utils import load_yaml_config as load_configuration`, but there's no advantage in doing that, either.

What I suggested was creating a main function that would load the configuration _and then call of the other functions_. That would be useful, because then the user would be able to create simulated images from simulation data without having to work out what order to call all of these functions in, or write their own code to so.

The reason why we originally suggesting splitting up the one big function you started with into a series of smaller, self contained functions wasn't that we thought that there shouldn't be a function that does the whole thing. The reason for splitting it up into multiple functions was to make the code easier to understand, test, maintain and resuse. There should still be a function that does the whole thing, it's just that instead of a great big block of code it should consist mostly of calls to the other functions.

Reply:
There are some inconsistencies between the functions of `mocks.py`. For instance functions `scale_light_by_distance` and `prepare_mocks` both use the distance to the galaxy to scale its light. 
The first step to create the main function is making all functions consistent.

Reply:
After making the functions consistent, the jupyter-notebook examples should be updated as the base for the main function that calls other functions and does all the steps to create the mock image.